tembo-cli/src/cmd/context.rs

```diff
@@ -1,27 +1,22 @@
-use crate::Result;
+use clap::{Args, Subcommand};
 
-use clap::ArgMatches;
-use simplelog::*;
+use set::ContextSetArgs;
 
 pub mod list;
 pub mod set;
 
-// handles all context command calls
-pub fn execute(args: &ArgMatches) -> Result<()> {
-    // execute the context subcommands
-    let res = match args.subcommand() {
-        Some(("list", sub_matches)) => list::execute(sub_matches),
-        Some(("set", sub_matches)) => set::execute(sub_matches),
-
-        _ => unreachable!(),
```

Comment:
this is an example why refactoring to use input validation of subcommands: the command "tembo context asdf" would just have panic on unreachable code. Instead, now it will show help like this:
```
error: unrecognized subcommand 'asdfasdf'

Usage: tembo context <COMMAND>

For more information, try '--help'.
```